src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -5701,131 +5700,145 @@ private ScanTypeArgumentListKind ScanTypeArgumentList(NameOptions options)
         }
 
         private ScanTypeFlags ScanPossibleTypeArgumentList(
-            ref SyntaxToken lastTokenOfList, out bool isDefinitelyTypeArgumentList)
+            out SyntaxToken greaterThanToken,
+            out bool isDefinitelyTypeArgumentList)
         {
             isDefinitelyTypeArgumentList = false;
+            Debug.Assert(this.CurrentToken.Kind == SyntaxKind.LessThanToken);
```

Comment:
view with whitepace off.

this was always called from a caller that already checked we were on a `<`.  So no need to check/indent all this code.